modular_citadel/code/modules/custom_loadout/custom_items.dm

```diff
@@ -439,3 +436,24 @@
 	icon_state = "moveralls"
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
 
+/obj/item/clothing/head/telegramhat
+	name = "Singing Telegram Hat"
+	icon = 'icons/obj/custom.dmi'
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
+	desc = "A red fez like hat, has a yellow lining on the side."
+	item_state = "telegramhat"
+	icon_state = "telegramhat"
+	armor = list("melee" = 0, "bullet" = -500, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0) //Dont shoot the messager!
+	mutantrace_variation = NO_MUTANTRACE_VARIATION
+
+/obj/item/clothing/under/telegramhat
+	name = "Singing Telegram Suit"
+	icon = 'icons/obj/custom.dmi'
```

Comment:
The icon is not the same one where the icon_state resides.

Reply:
Dose this fix it?